kura/distrib/src/main/resources/intel-up2-centos-7-nn/snapshot_0.xml

```diff
@@ -198,4 +198,32 @@
             </esf:property>
         </esf:properties>
     </esf:configuration>
+    <esf:configuration pid="org.eclipse.kura.http.server.manager.HttpService">
+        <esf:properties>
+            <esf:property array="false" encrypted="false" name="https.port" type="Integer">
+                <esf:value>443</esf:value>
+            </esf:property>
+            <esf:property array="false" encrypted="false" name="https.enabled" type="Boolean">
+                <esf:value>true</esf:value>
+            </esf:property>
+            <esf:property array="false" encrypted="false" name="http.enabled" type="Boolean">
```

Comment:
Breaking change? Especially with the self-signed certificate that browsers by default don't like, much.

Reply:
With the new commit(s) the HTTP service starts without restarting Kura, but doesn't work:
```
o.e.j.s.HttpChannel - /admin
java.lang.IllegalStateException: CookieConfig cannot be set after ServletContext is started
        at org.eclipse.jetty.server.session.SessionHandler$CookieConfig.setHttpOnly(SessionHandler.java:1464)
        at org.eclipse.kura.web.SecureBasicHttpContext.login(SecureBasicHttpContext.java:155)
...
```
In the HTTP case not even page reload works, so effectively Kura needs to be reinstalled, again.